File: app/src/main/java/com/mathroda/dashcoin/core/util/MyYAxisValueFormatter.java
package com.mathroda.dashcoin.core.util;


import com.github.mikephil.charting.components.YAxis;
import com.github.mikephil.charting.formatter.ValueFormatter;

import java.text.DecimalFormat;

public class MyYAxisValueFormatter extends ValueFormatter {

    private DecimalFormat mFormat;

    public MyYAxisValueFormatter () {
        mFormat = new DecimalFormat("###,###,##0.0"); // use one decimal
    }

    @Override
    public String getFormattedValue(float value) {
        return "$"+mFormat.format(value) ;
    }


}

File: app/src/main/java/com/mathroda/dashcoin/core/util/MyYAxisValueFormatter.kt
package com.mathroda.dashcoin.core.util

import com.github.mikephil.charting.formatter.ValueFormatter
import java.text.DecimalFormat

class MyYAxisValueFormatter : ValueFormatter() {
    private val mFormat: DecimalFormat = DecimalFormat("###,###,##0.0")
    override fun getFormattedValue(value: Float): String {
        return "$" + mFormat.format(value.toDouble())
    }


}